src/cleanvision/issue_managers/duplicate_issue_manager.py

```diff
@@ -24,6 +24,12 @@ def get_hash(image: Image.Image, params: Dict[str, Any]) -> str:
         return str(imagehash.whash(image, hash_size=hash_size))
     elif hash_type == "phash":
         return str(imagehash.phash(image, hash_size=hash_size))
+    elif hash_type == "ahash":
+        return str(imagehash.average_hash(image, hash_size=hash_size))
+    elif hash_type == "dhash":
+        return str(imagehash.dhash(image, hash_size=hash_size))
+    elif hash_type == "chash":
+        return str(imagehash.colorhash(image, binbits=hash_size))            
     else:
         raise ValueError("Hash type not supported")
 
```

Comment:
think you also need to update this file:

https://github.com/cleanlab/cleanvision/blob/606cdb059281c34bbb2f646a3534d13d17234f05/docs/source/tutorials/tutorial.ipynb#L513

where it says:  "Currently `whash` and `phash` are the supported hash types."

Reply:
Thanks, I've updated my code